File: C C++/Ago-31-2021/ex005.c
#include<stdio.h>

int main(){
    float n1,n2,n3,media;

    printf("Primeira nota: ");
    scanf("%f", &n1);
    printf("Segunda nota: ");
    scanf("%f", &n2);
    printf("Terceira nota: ");
    scanf("%f", &n3);

    media = (n1+n2+n3)/3;

    printf("\nA média das notas é: %.2f", media);
    return 0;
}

File: _java/ex005.java
//Programa cálculo média de 3 notas

package Uniso._java;
import java.util.Scanner;

public class ex005 {
    public static void main(String[] args) {
        Scanner scanf = new Scanner(System.in);
        float n1, n2, n3, media;

        System.out.print("Primeira nota: ");
        n1 = scanf.nextFloat();
        System.out.print("Segunda nota: ");
        n2 = scanf.nextFloat();
        System.out.print("Terceira nota: ");
        n3 = scanf.nextFloat();

        media = (n1 + n2 + n3)/3;

        System.out.println("\nA média das notas é "+ media);
        scanf.close();
    }
}
